### DISCUSSION TITLE
Development

### DISCUSSION THREAD

**Discussion by Hayden-Badenhop** [CarolineChief/Tafe240261#11] [General]
Wanted top let you guys know, I am now merging my feature to the development branch ( after i fix the conflicts ).

**Comment by Hayden-Badenhop** [comment]
I have fixed the conflicts, and pushed a working version of the development branch. Let me know if anyone has any problems

**Reply by CarolineChiefari** [reply]
Hey, I just pulled the latest version and can no longer get the calculator to work or even open on VS. Is it still working for you and have you pulled the feature_mortgage branch yet?
